app/forms/assessor_interface/assessment_declaration_decline_form.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class AssessorInterface::AssessmentDeclarationDeclineForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  attr_accessor :session, :assessment
+  attribute :declaration, :boolean
+  attribute :recommendation_assessor_note, :string
+
+  validates :declaration, presence: true
+  validates :recommendation_assessor_note,
+            presence: true,
+            if: -> { selected_failure_reasons_empty(assessment) }
```

Comment:
I wonder if this should be a method on the `assessment` model so it can be tested on its own. Perhaps `assessment.selected_failure_reasons_empty?`.

Reply:
I've move it and think i'm testing adequately